cirq-core/cirq/transformers/routing/route_circuit_cqc.py

```diff
@@ -25,30 +25,28 @@
 if TYPE_CHECKING:
     import cirq
 
-QidPair = Tuple['cirq.Qid', 'cirq.Qid']
+QidIntPair = Tuple[int, int]
 
 
-def disjoint_nc2_combinations(qubit_pairs: List[QidPair]) -> List[Tuple[QidPair, ...]]:
+def disjoint_nc2_combinations(
+    qubit_pairs: Sequence[QidIntPair],
+) -> List[Tuple[QidIntPair, QidIntPair]]:
     """Gets disjoint pair combinations of qubits pairs.
 
     For example:
 
-        >>> q = cirq.LineQubit.range(5)
+        >>> q = [*range(5)]
         >>> disjoint_swaps = cirq.transformers.routing.route_circuit_cqc.disjoint_nc2_combinations(
         ...     [(q[0], q[1]), (q[2], q[3]), (q[1], q[4])]
         ... )
```

Comment:
Update doctest input?

Reply:
q is now `[*range(5)]` instead of a list of qubits.